perfkitbenchmarker/providers/gcp/gcp_managed_relational_db.py

```diff
@@ -124,7 +133,7 @@ def _Delete(self):
     deleted.
     """
     cmd = util.GcloudCommand(self, 'sql', 'instances', 'delete',
-                             self.instance_id)
+                             self.instance_id, '--quiet')
```

Comment:
Would --quiet also stop logging for the delete operation? If so, is there a different option that would skip the confirmation but still do the logging? (like -y or such)